tidb-cloud/migrate-from-aurora-bulk-import.md

```diff
@@ -5,34 +5,60 @@ summary: Learn how to migrate data from Amazon Aurora MySQL to TiDB Cloud in bul
+This document describes how to migrate data from Amazon Aurora MySQL to TiDB Cloud in bulk using the import tools on TiDB Cloud console.
 
 ## Learn how to create an import task on the TiDB Cloud console
 
 To import data, perform the following steps:
 
 1. Navigate to the **Clusters** page.
-2. Locate your target cluster, and click **Import Data** in the upper-right corner of the cluster area. The **Data Import Task** page is displayed.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your target cluster on the **Clusters** page and click **Import Data** in the upper-right corner.
+2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
 
 3. Prepare source data according to [Learn how to create an Amazon S3 Bucket and prepare source data files](#learn-how-to-create-an-amazon-s3-bucket-and-prepare-source-data-files). You can see the advantages and disadvantages of different **Data Format** in the preparing data part.
-4. Fill in the **Data Source Type**, **Bucket URL**, and **Data Format** fields according to the specification of your source data.
+
+4. Select or fill in the **Data Format**, **Location**, **Bucket URL**, and **Role-ARN** fields according to the specification of your source data. For more information about how to create the bucket policy and role for cross-account access, see[Learn how to configure cross-account access](#learn-how-to-configure-cross-account-access).
+
 5. Fill in the **Username** and **Password** fields of the **Target Cluster**.
-6. Create the bucket policy and role for cross-account access according to [Learn how to configure cross-account access](#learn-how-to-configure-cross-account-access).
-7. Click **Import**.
 
-    A warning message about the database resource consumption is displayed.
+6. Modify the file patterns and add the table filter rules if needed. Click **Next**.
+
+    - **File Pattern**: modify the file pattern if you want to import files whose filenames match a certain pattern to a single target table.
+
+        > **Note:**
+        >
+        > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
+
+        To modify the file pattern, specify a custom mapping rule between CSV files and a single target table in the following fields:
+
+        - **File name**: enter a pattern that matches the names of the files to be imported. If you have one file only, enter the file name here directly.
+
+            For example:
 
-8. Click **Confirm**.
+            - `my-data?.sql`: all CSV files starting with `my-data` and one character (such as `my-data1.sql` and `my-data2.sql`) will be imported into the same target table.
```

Comment:
Please check if "CSV files" needs to be updated. According to [Migrate from Amazon Aurora MySQL to TiDB Cloud in Bulk | PingCAP Docs](https://docs.pingcap.com/tidbcloud/migrate-from-aurora-bulk-import#option-2-prepare-source-data-files-using-amazon-aurora-snapshots), the formats could be Dumpling or Aurora Snapshot.